spec/controllers/concerns/omniauth_login_spec.rb

```diff
@@ -530,15 +478,16 @@
 
     describe "Error occurs during authentication" do
       it "redirects to login page" do
```

Comment:
Now I know this is not because of this PR because this is but a refactoring, but are you perhaps in the right context still to address the issue that this spec here clearly isn't actually doing what it claims it's doing?

It says on error this should redirect to the signin page and show the actual error in the flash message (although the latter part is not tested).

What actually happens, though, is that you get an internal server error on the callback route without a redirect and without a useful error message.

The actual error is logged correctly at least, but that's not useful for end users.

![image](https://github.com/user-attachments/assets/a4da4e5a-7d02-4db8-bbb5-55997676ae05)

Reply:
Actually, the PR does add the output of the error message!

Reply:
Good catch, thanks 👍